includes/wikia/models/DesignSystemGlobalFooterWikiModel.class.php

```diff
@@ -0,0 +1,104 @@
+<?php
+
+class DesignSystemGlobalFooterWikiModel extends DesignSystemGlobalFooterModel {
+
+	protected $wikiId;
+
+	public function __construct( $id, $lang = self::DEFAULT_LANG ) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Use "static" keyword instead of "self". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS2037)